bin/pycbc_multi_inspiral

```diff
@@ -142,6 +142,39 @@ logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
 inj_filter_rejector = pycbc.inject.InjFilterRejector.from_cli_multi_ifos(opt,opt.instruments)
 ctx = scheme.from_cli(opt)
 
+def network_chisq(chisq, chisq_dof, snr_dict):
```

Comment:
Check that the chisq-dof here is *actually* the number of frequency bins, and not the actual DOF.

Reply:
Looking through the singleDetPowerChisq code here, I think it actually is the DOF that is output. Changing the code now.